e2e-pw/src/oss/fixtures/loader.ts

```diff
@@ -126,7 +126,9 @@ export class OssLoader extends AbstractFiftyoneLoader {
       await forceDatasetFromSelector();
     } else {
       await page.goto(`/datasets/${datasetName}`);
-      const location = await page.evaluate(() => window.location.href);
+      const location = await page.evaluate(() =>
+        window.location.href.replace(/view=*/, "")
```

Comment:
I'm confused why we're doing this. Maybe add a comment here explaining why we're replacing the view query param (and not other params)?